lib/app/router.js

```diff
@@ -11,6 +11,9 @@ import Router from 'vue-router'
     res += (route.component) ? ',\n\t' + tab + 'component: ' + (splitChunks.pages ? route._name : `() => ${route._name}.default || ${route._name}`) : ''
     res += (route.redirect) ? ',\n\t' + tab + 'redirect: ' + JSON.stringify(route.redirect) : ''
     res += (route.meta) ? ',\n\t' + tab + 'meta: ' + JSON.stringify(route.meta) : ''
+    res += (typeof route.props !== 'undefined') ? ',\n\t' + tab + 'props: ' + (typeof route.props === 'function' ? serialize(route.props) : JSON.stringify(route.props)) : ''
```

Comment:
For consistency, this may be simplified into `(route.props)`

Reply:
According to vue-router doc props can have Boolean value, so we can't just check its falsy value

Reply:
@farnabaz Regardless, `route.props != null` is the canonical way of checking `null`/`undefined` values in javascript. According to clean-code rules, checking for positive instead of negative is considered slightly easier to read. Robert C. Martin

If doesn't appear to be anywhere where `null` is accepted instead of `undefined`, so the code should be:

```js
res += route.props == null ? '' : ',\n\t' + tab + 'props: ' + (typeof route.props === 'function' ? serialize(route.props) : JSON.stringify(route.props))
```
Right?

Reply:
@dotnetCarpenter You're right. I'll create a new pull request to cleanup my code.
Thank you very much for your comment :)